### PR TITLE
reduce multiple empty lines to 1

### LINKED ISSUE #1: patch multiple-empty-lines to error and disallow more than one
This will eliminate duplicate lines in our services. 

I could see how this would be breaking for older services, should it be a version number change instead of a patch?

### PR DESCRIPTION
**Pull request by generalandrew**
also add end of file and beginning of file new line restrictions

closes #1

### CODE REVIEW

**Comment by generalandrew** [comment]
Updated this lint to be a `warning` instead of an `error` to eliminate chances of breakage. This also allows for multi line breaks just in case it improves readability.

File: index.js [hunk 1]
```diff
@@ -28,7 +28,7 @@ module.exports = {
         "indent": ["error", 4, { "SwitchCase": 1 }],
         "no-console": 1,
         "no-debugger": 2,
-        "no-multiple-empty-lines": 2,
+        "no-multiple-empty-lines": ["warn", { "max": 1, "maxEOF": 0, "maxBOF": 0 }],
```
Comment (andrewburgess) [inline index.js]: It should be an error. There aren't any cases where multiple newlines should be used in a code base. Additionally, we want one empty newline at the end of files. Does this allow for that?

http://stackoverflow.com/questions/5813311/no-newline-at-end-of-file

Comment (generalandrew) [inline index.js]: For whatever reason, setting end of file to 0 allows for 1 new line at the end, setting for 1 allows for 2 newlines at the end. I'll change it back to an error.

Comment (andrewburgess) [inline index.js]: Ok, I think it means max number of additional newlines. Just wanted to make sure it allowed for a single one at the end

Comment (generalandrew) [inline index.js]: maxEOF: 0 in action

**Pull request merged**